integration/js/script/test-setup

```diff
@@ -5,10 +5,20 @@ const fs = require('fs');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const path = require('path');
 
-const test_config_path = path.join(__dirname, '..', '..', 'configs', 'messaging_session_test.config.json');
+var myArgs = process.argv.slice(2);
```

Comment:
Since, you are an expert on this, would be glad if you can add documentation on what this script does :)

Reply:
Actually can we revert this change and remove the messaging test?

Reply:
Removed and added documentation